modules/end-user-guide/partials/proc_building-an-image-for-an-ide-based-on-the-intellij-platform.adoc

```diff
@@ -0,0 +1,160 @@
+[id="building-an-image-for-an-ide-based-on-the-intellij-platform_{context}"]
+= Building an image for an IDE based on the IntelliJ Platform
+
+This procedure describes how to build an image for an IDE based on the IntelliJ Platform up to version 2020.3. See the link:https://github.com/che-incubator/jetbrains-editor-images/blob/main/doc/Compatible-IDE.md[list of compatible IDEs].
```

Comment:
:) Now you can see one more benefit of my suggestion for `IntelliJ Platform-based IDEs` - the current line reads "_based on the **IntelliJ Platform up to version 2020.3**_".

Reply:
```suggestion
This section describes how to build images for link:https://plugins.jetbrains.com/docs/intellij/intellij-platform.html#ides-based-on-the-intellij-platform[IntelliJ Platform-based IDEs] up to version 2020.3.
```

Reply:
Why remove the link to the list of compatible IDEs?

Reply:
@themr0c Apologies, this is clearly a misplaced copy-paste op.

Reply:
@themr0c Sorry about that, a copy-paste gone wrong on the suggestion.

Reply:
Discarding this suggestion as we have clearly established that  JetBrains IDE s are only a subset of all IDEs based on the IntelliJ Platform that are supported (Android Studio).

Reply:
@themr0c I believe it's not possible to mention _**IDE based on the IntelliJ Platform up to version 2020.3**_ for 2 reasons:
1. Only JetBrains IDEs have versions 2020.3. Non-JetBrains IDEs most likely have other versioning.
2. **_version 2020.3_** is the version of the IDEs, not of the IntelliJ Platform.

Reply:
@themr0c In other words, the way I understand this, where you mention 2020.3 you are referring to JetBrains IDEs.